assets/src/edit-story/components/panels/document/slug/slug.js

```diff
@@ -68,9 +88,7 @@ function SlugPanel() {
     [updateStory]
   );
 
-  const handleChange = useCallback((value) => updateSlug(value, true), [
-    updateSlug,
-  ]);
+  const handleChange = useCallback((evt) => setSlug(evt.target.value), []);
 
   const handleBlur = useCallback((evt) => updateSlug(evt.target.value, false), [
     updateSlug,
```

Comment:
I currently have this only updating the saved slug when we blur. Do we want to save this every single key stroke?

Reply:
Only on blur sounds good!